js/privatebin.js

```diff
@@ -591,6 +591,32 @@ jQuery.PrivateBin = (function($, RawDeflate) {
             return expirationDate;
         };
 
+        /**
+         * Convert Bytes to KB/MB/GB
```

Comment:
As you noted, currently PrivateBin reports all sizes in binary bytes, not SI-units. We can have a discussion if these days one or the other is more prevalent - I'm a Linux user and all my CLI and GUI tools display binary bytes and SI-ones have to requested by switch, so to me those SI-units cause great confusion.

But whatever we do, we should stay consistent and use the same units in all messages. I would vote on keeping binary units.